app/page.tsx

```diff
@@ -39,5 +43,11 @@ export default function Home() {
           </ul>
         </div>
     </section>
+    <section className="max-w-4xl py-12 mx-auto px-4 md:px-8">
```

Comment:
Probably. I was experimenting with adding eslint but was thinking it would probably be better if it was left to you guys to do as I don't know what your preferred options are.